test/1_test_fullpath.js

```diff
@@ -897,26 +897,24 @@ contract('Voucher tests', async (addresses) => {
       );
 
       //Check VoucherKernel state
-      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+      const voucherStatus = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
-      assert.isTrue(voucherStatus.status.eq(new BN(192)));
+      assert.isTrue(voucherStatus[0].eq(new BN(192)));
 
       const transaction = await web3.eth.getTransaction(txRedeem.tx);
       const transactionBlock = await web3.eth.getBlock(transaction.blockNumber);
-      assert.isTrue(
-        voucherStatus.complainPeriodStart.eq(new BN(transactionBlock.timestamp))
-      );
+      assert.isTrue(voucherStatus[3].eq(new BN(transactionBlock.timestamp)));
     });
 
     it('mark non-redeemed voucher as expired', async () => {
-      const statusBefore = await contractVoucherKernel.vouchersStatus(
+      const statusBefore = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey2
       );
 
       // [1000.0000] = hex"80" = 128 = COMMITTED
       assert.equal(
-        web3.utils.toHex(statusBefore.status),
+        web3.utils.toHex(statusBefore[0]),
```

Comment:
Please use constants instead of hard-coded indexes for statusBefore (same as remark above)

Reply:
Remaining hard-coded indexes at line 975